account_asset_disposal/models/account_asset_asset.py

```diff
@@ -55,7 +55,9 @@ def _disposal_line_loss_prepare(self, date, loss_account):
             'name': _('Asset loss'),
             'journal_id': self.category_id.journal_id.id,
             'account_id': loss_account.id,
-            'analytic_account_id': self.category_id.account_analytic_id.id,
+            'analytic_account_id': (
```

Comment:
Yo aquí añadirira un if preguntando si existe el atributo:
self.account_analytic_id.id 
if hasattr(self, 'account_analytic_id') and self.account_analytic_id.id else self.category_id.account_analytic_id.id

y quitar la dependencia con el modulo